examples/kitchen-sink.yaml

```diff
@@ -91,6 +91,12 @@ logger_provider:
             #
             # Environment variable: OTEL_EXPORTER_OTLP_INSECURE, OTEL_EXPORTER_OTLP_LOGS_INSECURE
             insecure: false
+    # Configure a simple span processor.
+    - simple:
+        # Configure exporter.
+        exporter:
+          # Configure exporter to be console.
+          console: {}
```

Comment:
Its consistent but the console exporter for logs can be tricky to avoid circular loops: 
Log framework -> Otel Log Bridge API -> Otel Log SDK -> Console Exporter -> Log Framework -> 

Maybe we don't need to worry about that for the sake of config. Its tricky to get right, but still possible to configure.

Reply:
Right, to be clear this was possible to configure even before this change :) we just didnt have a `console` section in the log record exporter called out specifically